---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Using scRNA-seq to Identify Transcriptional Variation in the Malaria Parasite
  Ookinete Stage
subtitle: ''
summary: ''
authors:
- Kathrin Witmer
- Farah Aida Dahalan
- Tom Metcalf
- Arthur M Talman
- Virginia M Howick
- Mara K N Lawniczak
tags: []
categories: []
date: '2021-01-01'
lastmod: 2021-03-01T17:22:25Z
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2021-03-01T17:22:24.988198Z'
publication_types:
- '2'
abstract: The crossing of the mosquito midgut epithelium by the malaria parasite motile
  ookinete form represents the most extreme population bottleneck in the parasite
  life cycle and is a prime target for transmission blocking strategies. However,
  we have little understanding of the clonal variation that exists in a population
  of ookinetes in the vector, partially because the parasites are difficult to access
  and are found in low numbers. Within a vector, variation may result as a response
  to specific environmental cues or may exist independent of those cues as a potential
  bet-hedging strategy. Here we use single-cell RNA-seq to profile transcriptional
  variation in Plasmodium berghei ookinetes across different vector species, and between
  and within individual midguts. We then compare our results to low-input transcriptomes
  from individual Anopheles coluzzii midguts infected with the human malaria parasite
  Plasmodium falciparum. Although the vast majority of transcriptional changes in
  ookinetes are driven by development, we have identified candidate genes that may
  be responding to environmental cues or are clonally variant within a population.
  Our results illustrate the value of single-cell and low-input technologies in understanding
  clonal variation of parasite populations.
publication: '*Front. Cell. Infect. Microbiol.*'
---
